ubuntudesign/developer.ubuntu.com
# Fix cards, remove unneeded contribute page

## Done

Fix card issues sitewide
Remove unneeded contribute page from community-resources

## QA

- Run ./run
- Go to:
 - http://0.0.0.0:8015/community-resources
 - http://0.0.0.0:8015/snapcraft
 - http://0.0.0.0:8015/core
- http://0.0.0.0:8015/core/tutorials
- Make sure there is no text underline on hover and that the date doesn’t crash into the title as is on the same pages on http://developer.ubuntu.com-v1-dev.demo.haus/

**Travis CI fix here vanilla-framework/vanilla-framework#1050**
